### PR TITLE
⚡Update Search_Widgets_NoResults

### PR DESCRIPTION
**Pull request by OfficerKD637**
Makes it maybe a little more appealing when you land on the Search or Discover hub?

I just used a very generic sentence for the textbox haha. You can customize it and add it to the strings.po file to help with the translations.

### CODE REVIEW

**Comment by jurialmunkey** [comment]
There's no need to duplicate these controls. Just use a label variable.

**Comment by OfficerKD637** [comment]
Jurial dropping these tips like...
salty-salt-bae

And just so I can do better in the future, what's the difference between the two approaches (duplicated controls vs using variables)? Is there an efficiency benefit or maybe something else? It's always such a  great opportunity to learn something new from you!

Thank you!

**Comment by jurialmunkey** [comment]
More efficient, better performance, easier code maintenance, simpler to extend.

Variable is more efficient as it provides early exit for conditions (it only evaluates until true). Separate controls means both evaluated for visibility, and both labels evaluated. It's also faster on initial window load to create one control rather than two.

Plus way simpler to maintain one code block if making a style change,.and much easier to add conditions or extend a variable in future rather than managing extra visibility conditions all over the place and trying to keep track of how each interacts 

Basically if a control is in the same place as another with the exact same style/dimensions and both will never be shown together, then you should be thinking of using a variable instead.
